### PR TITLE
Added Playground for different languages

### PR DESCRIPTION
**Pull request by Ajay-singh1**
## What does this PR do?
Add resource(s) 

## For resources
Playground
### Description
Added playground for Various Languges.
### Why is this valuable (or not)?
Yes
### How do we know it's really free?
It is free
### For book lists, is it a book? For course lists, is it a course? etc.
Playground
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: more/free-programming-playgrounds.md [hunk 1]
```diff
@@ -92,9 +92,20 @@
 
 ### <a id="csharp"></a>C\#
 
+* [Codeapi](https://codapi.org/csharp)
+* [CodeSandbox](https://codesandbox.io/s/c-z2m9v)
+* [Dotfiddlenet](https://dotnetfiddle.net)
+* [GDB](https://www.onlinegdb.com/online_csharp_compiler)
```
[the remark this answers is not in the document]
Comment (eshellman) [reply more/free-programming-playgrounds.md:98]: also, a duplicate. This is why we want to use name from source.

File: more/free-programming-playgrounds.md [hunk 2]
```diff
@@ -92,9 +92,20 @@
...
+* [Getvm](https://getvm.io/playgrounds/c-sharp-playground)
+* [Jdoodle](https://www.jdoodle.com/compile-c-sharp-online)
+* [myCompiler](https://www.mycompiler.io/online-csharp-compiler)
```
Comment (eshellman) [inline more/free-programming-playgrounds.md:101]:
```suggestion
* [Online C# Compiler](https://www.mycompiler.io/online-csharp-compiler) - myCompiler
```

File: more/free-programming-playgrounds.md [hunk 3]
```diff
@@ -92,9 +92,20 @@
...
+* [Onecompiler](https://onecompiler.com/csharp/3wv9zujyk)
 * [OnlineGDB](https://www.onlinegdb.com/online_csharp_compiler)
-* [Programiz - Online Compiler](https://www.programiz.com/csharp-programming/online-compiler/)
+* [Programiz - Online Compiler](https://www.programiz.com/csharp-programming/online-compiler)
+* [Scholarhat](https://www.scholarhat.com/compiler/csharp)
```
Comment (eshellman) [inline more/free-programming-playgrounds.md:105]:
```suggestion
* [C# Online Compiler](https://www.scholarhat.com/compiler/csharp) - ScholarHat
```

File: more/free-programming-playgrounds.md [hunk 4]
```diff
@@ -92,9 +92,20 @@
...
+* [Sharplab](https://sharplab.io)
```
Comment (eshellman) [inline more/free-programming-playgrounds.md:106]:
```suggestion
* [SharpLab](https://sharplab.io)
```

File: more/free-programming-playgrounds.md [hunk 5]
```diff
@@ -92,9 +92,20 @@
...
 * [SoloLearn](https://code.sololearn.com/csharp)
+* [W3Schools](https://www.w3schools.com/cs/cs_compiler.php)
```
Comment (eshellman) [inline more/free-programming-playgrounds.md:108]:
```suggestion
* [C# Online Compiler](https://www.w3schools.com/cs/cs_compiler.php) - W3Schools
```

File: more/free-programming-playgrounds.md [hunk 6]
```diff
@@ -164,7 +175,9 @@
 
 ### Git
 
-* [Git](https://learngitbranching.js.org/?NODEMO)
+* [Codeapi](https://git-school.github.io/visualizing-git)
```
Comment (eshellman) [inline more/free-programming-playgrounds.md:178]:
```suggestion
* [Visualizing Git](https://git-school.github.io/visualizing-git)
```

File: more/free-programming-playgrounds.md [hunk 7]
```diff
@@ -204,6 +217,7 @@
 
 ### Ionic
 
+* [Codepen](https://codepen.io/ajp8164/pen/zrLQYP)
```
Comment (eshellman) [inline more/free-programming-playgrounds.md:220]: looks like sample code, not a playground as such

**Comment by ehis-042** [comment]
Please what is the problem of this project

**Comment by Ajay-singh1** [comment]
@ehis-042 I will fix this today!

File: more/free-programming-playgrounds.md [hunk 8]
```diff
@@ -92,9 +92,20 @@
...
+* [CodeSandbox](https://codesandbox.io/s/c-z2m9v)
+* [Dotfiddlenet](https://dotnetfiddle.net)
+* [GDB](https://www.onlinegdb.com/online_csharp_compiler)
+* [Getvm](https://getvm.io/playgrounds/c-sharp-playground)
+* [Jdoodle](https://www.jdoodle.com/compile-c-sharp-online)
+* [myCompiler](https://www.mycompiler.io/online-csharp-compiler)
+* [Onecompiler](https://onecompiler.com/csharp/3wv9zujyk)
 * [OnlineGDB](https://www.onlinegdb.com/online_csharp_compiler)
-* [Programiz - Online Compiler](https://www.programiz.com/csharp-programming/online-compiler/)
+* [Programiz - Online Compiler](https://www.programiz.com/csharp-programming/online-compiler)
```
[the remark this answers is not in the document]
Comment (eshellman) [reply more/free-programming-playgrounds.md:102]: Why are you changing it?

File: more/free-programming-playgrounds.md [hunk 9]
```diff
@@ -164,7 +171,9 @@
 
 ### Git
 
-* [Git](https://learngitbranching.js.org/?NODEMO)
+* [Git-school](https://git-school.github.io/visualizing-git)
+* [Learn Git Branching](https://learngitbranching.js.org/?NODEMO)
+* [Visualizing Git](https://git-school.github.io/visualizing-git)
```
Comment (eshellman) [inline more/free-programming-playgrounds.md:176]: the resource on lines 174 and line 176 are duplicates

**Review by eshellman — CHANGES_REQUESTED** [review]
still have duplcate resources!

**Comment by Ajay-singh1** [comment]
> still have duplcate resources!

I have to delete it and re-add it because the lint was failing

**Review by HSUYATANG — COMMENTED** [review]
@@ -92,8 +92,15 @@
 
 ### C#
 
+* [C# Online Compiler](https://www.scholarhat.com/compiler/csharp) - ScholarHat
+* [C# Online Compiler](https://www.w3schools.com/cs/cs_compiler.php) - W3Schools
+* [C# playground](https://codapi.org/csharp) - Codeapi
+* [OneCompiler](https://onecompiler.com/csharp/3wv9zujyk)
+* [Online C# Compiler](https://www.mycompiler.io/online-csharp-compiler) - myCompiler
+* [Online C# Compiler IDE](https://www.jdoodle.com/compile-c-sharp-online) - JDoodle
 * [OnlineGDB](https://www.onlinegdb.com/online_csharp_compiler)
-* [Programiz - Online Compiler](https://www.programiz.com/csharp-programming/online-compiler/)
+* [Programiz - Online Compiler](https://www.programiz.com/csharp-programming/online-compiler)
+* [SharpLab](https://sharplab.io)
 * [SoloLearn](https://code.sololearn.com/csharp)
 
@@ -164,7 +171,8 @@
 
 ### Git
 
-* [Git](https://learngitbranching.js.org/?NODEMO)
+* [Git-school](https://git-school.github.io/visualizing-git)
+* [Learn Git Branching](https://learngitbranching.js.org/?NODEMO)
 
 ### Go

**Pull request merged**